modules/serverless-configuring-kube-rbac-proxy-resources-for-eventing.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * eventing/kube-rbac-proxy-eventing.adoc
+
+:_content-type: REFERENCE
+[id="serverless-configuring-kube-rbac-proxy-resources-for-eventing_{context}"]
+= Configuring kube-rbac-proxy resources for Eventing
+
+You can globally override resource allocation for the `kube-rbac-proxy` container by using the {ServerlessOperatorName} CR.
+
+[NOTE]
+----
+You can also override resource allocation for a specific deployment.
+----
+
+The following configuration sets Knative Eventing `kube-rbac-proxy` minimum and maximum CPU and memory allocation:
+
+.KnativeEventing CR example
+[source,yaml]
+----
+apiVersion: eventing.knative.dev/v1beta1
+kind: KnativeEventing
+metadata:
+  name: ks
```

Comment:
Let's use `name: knative-eventing` to align this with the rest of documentation.